### ISSUE TITLE
Inconsistent accessibility error in REPL

### ISSUE THREAD

**Issue by mspielberg** [ManlyMarco/RuntimeUnityEditor#86]
Running RUE 5.5.1 in UMM 0.31.0.0, and I am not seeing the "Failed to initialize REPL environment" error in the logs, but I *am* getting the "CS0060: Inconsistent accessibility" errors in the REPL. The target game is Derail Valley which runs on the .NET Framework 4.x version of the Unity runtime.

I looked at #68 but opened a new issue due to the lack of the initialization failure message. Any other known causes for this issue?

**Comment by ManlyMarco** [comment]
Should be resolved in upcoming 6.0 release. Reopen the issue if it's not fixed.
